dss/util/auth/authorize.py

```diff
@@ -0,0 +1,23 @@
+class Authorize:
+    """ abstract class for authorization classes"""
+    def __init__(self):
+        pass
+
+    def security_flow(self, authz_methods, *args, **kwargs):
```

Comment:
As per other comment, the `authz_methods` parameter could use some explanation in the docstring